### ISSUE TITLE
New "Enable Services" Feature broken 

### ISSUE THREAD

**Issue by gardotd426** [oscartbeaumont/ElectronPlayer#57]
I just updated to the latest version of electronplayer on Arch Linux. I was sad when I saw that some services appeared to have been removed, but then when I clicked on the menu, I saw that actually services had been added, they just needed to be enabled in Settings > Enabled Services. So I enabled Amazon Prime, which worked just fine. However, nothing else does. The only services that show up now matter how many times I select additional services are Netflix, Youtube, and Amazon Prime. Hulu will not show up no matter what, neither will Floatplane. However, if I select "Custom URL" and type in hulu.com, it works and goes to Hulu. 

After enabling every service in the menu to see which ones worked and which ones didn't, and restarting electronplayer, here are the results:

- Netflix: Present
- Youtube: Present
- Twitch: Present
- Amazon Prime: Present
- CBS All Access: Present
- Vudu: Present
- Disney+: Present
- YoutubeTV: Present
- Hulu: Not Present
- Floatplane: Not Present

**Comment by oscartbeaumont** [comment]
Yes, this is an issue. I am struggling to find time to maintain this project so it hasn't been fixed yet. I will do my best to fix it as soon as I can.

**Comment by gardotd426** [comment]
Hey no worries man, I wasn't complaining, just trying to help out since it wasn't listed as a known issue from what I saw, y'know? I wish I knew any programming or I'd offer to help, This is one of the apps I first download on every Linux system I install, I love it. Hell I've heard electron is supposed to be easier than some, I might see if I can't learn some so I can help
